CMakeLists.txt

```diff
@@ -51,37 +51,42 @@ target_include_directories(idl_messages INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/au
 
 
 
-# Add the yarp device that interacts with YARP motor interface via C++ 
-# and with IHMC-ORS software with DDS .idl messages 
+# Add the yarp device that interacts with YARP motor interface via C++
+# and with IHMC-ORS software with DDS .idl messages
 set(BUILD_SHARED_LIBS ON)
 yarp_configure_plugins_installation(ihmc-ors-yarp)
-set(YARP_DEVICE_NAME bridgeIHMCORS)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
-yarp_prepare_plugin(${YARP_DEVICE_NAME} TYPE ${YARP_DEVICE_NAME}
-                                        INCLUDE ${YARP_DEVICE_NAME}.h
+set(YARP_DEVICE_NAME bridge_ihmc_ors)
```

Comment:
Remember to change also the content of the `.ini` file, otherwise it will not work.